src/logic/cmp.rs

```diff
@@ -64,17 +115,8 @@ impl ApInt {
         }
     }
 
-    /// Unsigned less-equals (`ule`) comparison between `self` and `rhs`.
```

Comment:
Also: Please re-add `# Errors` doc-sections everywhere!
It is useful to have in documentation and is recommended in the general Rust doc guidelines.

Reply:
https://rust-lang-nursery.github.io/api-guidelines/documentation.html